src/mesh.cpp

```diff
@@ -100,9 +114,22 @@ void readMainFile(Mesh &mesh, const std::string &filename, const std::string &da
     mesh.data.resize(NumPoints * dim, 0.0);
   }
 
-  /*
-  !!Add Mesh Connecivity information in next PR!!
-  */
+  for (int i = 0; i < reader->GetUnstructuredGridOutput()->GetNumberOfCells(); i++) {
+    int cellType = reader->GetUnstructuredGridOutput()->GetCell(1)->GetCellType();
+
+    if (cellType == 5) { // Triangular Element
```

Comment:
Isn't there a macro that defines the type?

Reply:
Yes, here, https://vtk.org/doc/nightly/html/vtkCellType_8h_source.html, definitely makes the code more human readable.